.github/workflows/zxc-build-library.yaml

```diff
@@ -76,7 +76,7 @@ jobs:
         run: mkdir -p "${{ github.workspace }}/b/vcpkg_cache"
 
       - name: Install CMake & Ninja
-        uses: lukka/get-cmake@latest
+        uses: lukka/get-cmake@v3.27.7
```

Comment:
Good for now but like I mentioned in the PR in the C++ protobufs repo, I'll try and loop this in into a monthly "update" PR, which can keep our dependencies and actions up-to-date.

Reply:
Sounds good to me :)